core/src/main/java/org/apache/accumulo/core/util/tables/TableMapping.java

```diff
@@ -108,7 +108,7 @@ public void rename(final TableId tableId, final String oldName, final String new
       throws InterruptedException, KeeperException, AcceptableThriftTableOperationException {
     if (namespaceId == Namespace.ACCUMULO.id()) {
       throw new AssertionError(
-          "Renaming built-in table maps in Accumulo namespace should not be possible");
+          "Renaming tables in the Accumulo namespace table map should not be possible");
```

Comment:
Another small fix for consistency.
```suggestion
          "Renaming built-in tables in the Accumulo namespace table map should not be possible");
```